css-backgrounds-4/Overview.bs

```diff
@@ -326,7 +326,7 @@ Painting Area: the 'background-clip' property</h3>
 	The syntax of the property is given with
 
 	<pre class=prod>
-	<dfn>&lt;bg-clip></dfn> = <<visual-box>> | border-area| text
+	<dfn>&lt;bg-clip></dfn> = <<visual-box>> | [ border-area || text ]
```

Comment:
The spec needs to explain what happens when 2 values are provided. Is the background clipped to the intersection? Union?